web/modules/custom/nlc_prototype/src/Controller/DirectoryTokenAccessConfirmController.php

```diff
@@ -173,19 +173,19 @@ protected function getCurrentUser() {
   public function build() {
 
     $build = [];
-    $format = filter_default_format();
-    $build['intro'] = [
-      '#type' => 'processed_text',
-      '#text' => $this->t('You will receive a secure link to your email address.'),
-      '#format' => $format,
-    ];
 
     $url = Url::fromRoute('nlc_prototype.directory.token_access');
-    $build['form_link'] = [
-      '#type' => 'link',
-      '#title' => $this->t('Go to form'),
-      '#url' => $url,
+    $build['intro'] = [
+      '#type' => 'inline_template',
+      '#context' => [
+        'text_one' => 'We have sent a secure link to XXXX@XXXX to log you in. Check your email.',
```

Comment:
This looks a bit odd to me — this looks like filler text that needs a token or something added to be replaced. Does this need to be checked?

Reply:
Yeah I couldn't figure out how to get the email submitted variable into this page. Do you know how?

Reply:
I'll have a go, see what I can do — I'll pull the branch locally and see if I can make any headway with  it